luarules/gadgets/game_team_power_watcher.lua

```diff
@@ -62,233 +29,462 @@ if gadgetHandler:IsSyncedCode() then
         {threshold = 475000, techLevel = 3.5},
         {threshold = 600000, techLevel = 4},
         {threshold = 725000, techLevel = 4.5}
-    }
+    },
+
+
+    --stores the values of powers for teams
+    PeakTeamPowers = {},
+    TeamPowers = {},
+
+    --stores the power of each living unit as a table {unitID, power, unitTeam}
+    UnitsWithPower = {},
+
+    
+    --teams
+    ScavengerTeam = nil,
+    RaptorTeam = nil,
+    NeutralTeam = nil,
+    AITeams = {},
+    HumanTeams = {}
```

Comment:
Why are these team IDs needed by the consumer of the API?

Reply:
Because it's unintuitive figuring out how to get them otherwise. Especially for BARb's, took me around 30 minutes to figure out how it's done elsewhere. This lets people get those values in here with 0 extra effort. Since I already have them, why not?

Reply:
An example I can think of is for dynamic evolution option for evocom, it might be useful to double or halve the power values of Raptors or Scavengers to further refine desired results in specific game modes.